docs/zdgbook/ObjectPublishing.rst

```diff
@@ -826,33 +826,31 @@ Method Arguments
 ----------------
 
 Sometimes you may wish to control which object is published based on
-form data. For example, you might want to have a form with a select
```

Comment:
You can leave the old line the way it is, it is correct

Reply:
Thanks! Reverted it. Also the the other line in this subsection (which you reviewed in the other pull request, as I falsely pushed this one commit to both pull requests).